src/simulation.hpp

```diff
@@ -2293,13 +2293,21 @@ template <typename problem_t> void AMRSimulation<problem_t>::InitPhyParticles()
 	// Read particle parameters from input file
 	quokka::particleParmParse();
 
+	if (quokka::particle_random_seed != 0) {
+		amrex::InitRandom(static_cast<unsigned long>(quokka::particle_random_seed)); // NOLINT(google-runtime-int)
```

Comment:
You need to set this to a different value on each MPI rank. You also need to set the GPU seed separately: https://github.com/AMReX-Codes/amrex/blob/6ab61fb6d8a752a5b586cc98619c6cc13e417394/Src/Base/AMReX_Random.cpp#L89C1-L89C11